### PR TITLE
zh-cn: tutorials/generative/style_transfer.ipynb

### PR DESCRIPTION
**Pull request by nounotabe**

### CODE REVIEW

File: site/zh-cn/tutorials/generative/style_transfer.ipynb [hunk 1]
```diff
@@ -0,0 +1,1164 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "klh6ObK2t_vH"
+      },
+      "source": [
+        "定义一个加载图像的函数，并将其最大尺寸限制为512像素。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]:
```suggestion
        "定义一个加载图像的函数，并将其最大尺寸限制为 512 像素。"
```

File: site/zh-cn/tutorials/generative/style_transfer.ipynb [hunk 2]
```diff
@@ -0,0 +1,1164 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GEwZ7FlwrjoZ"
+      },
+      "source": [
+        "## 定义内容和风格的表示\n",
+        "\n",
+        "使用模型的中间层来获取图像的*内容*和*风格*表示。 从网络的输入层开始，前几个层的激励响应表示边缘和纹理等低级特征。 随着层数加深，最后几层代表更高级的特征——实体的部分，如*轮子*或*眼睛*。 在此教程中，我们使用的是VGG19网络结构，这是一个已经预训练好的图像分类网络。 这些中间层是从图像中定义内容和风格的表示所必需的。 对于一个输入图像，我们尝试匹配这些中间层的相应风格和内容目标的表示。\n"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]:
```suggestion
        "使用模型的中间层来获取图像的*内容*和*风格*表示。 从网络的输入层开始，前几个层的激励响应表示边缘和纹理等低级特征。 随着层数加深，最后几层代表更高级的特征——实体的部分，如*轮子*或*眼睛*。 在此教程中，我们使用的是 VGG19 网络结构，这是一个已经预训练好的图像分类网络。 这些中间层是从图像中定义内容和风格的表示所必需的。 对于一个输入图像，我们尝试匹配这些中间层的相应风格和内容目标的表示。\n"
```

File: site/zh-cn/tutorials/generative/style_transfer.ipynb [hunk 3]
```diff
@@ -0,0 +1,1164 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ljpoYk-0f6HS"
+      },
+      "source": [
+        "现在，加载没有分类部分的`VGG19`，并列出各层的名称"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]:
```suggestion
        "现在，加载没有分类部分的 `VGG19`，并列出各层的名称："
```

File: site/zh-cn/tutorials/generative/style_transfer.ipynb [hunk 4]
```diff
@@ -0,0 +1,1164 @@
...
+      "source": [
+        "## 建立模型 \n",
+        "\n",
+        "使用`tf.keras.applications`中的网络可以让我们非常方便的利用Keras的功能接口提取中间层的值。\n",
+        "\n",
+        "在使用功能接口定义模型时，我们需要指定输入和输出：\n",
+        "\n",
+        "`model = Model(inputs, outputs)`\n",
+        "\n",
+        "以下函数构建了一个VGG19模型，该模型返回一个中间层输出的列表："
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]:
```suggestion
        "以下函数构建了一个 VGG19 模型，该模型返回一个中间层输出的列表："
```

File: site/zh-cn/tutorials/generative/style_transfer.ipynb [hunk 5]
```diff
@@ -0,0 +1,1164 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "lGUfttK9F8d5"
+      },
+      "source": [
+        "## 风格计算\n",
+        "\n",
+        "图像的内容由中间特征图的值表示。\n",
+        "\n",
+        "事实证明，图像的风格可以通过不同特征图上的平均值和相关性来描述。 通过在每个位置计算特征向量的外积，并在所有位置对该外积进行平均,可以计算出包含此信息的Gram矩阵。 对于特定层的Gram矩阵，具体计算方法如下所示：\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]: 中英文需用空格分隔

File: site/zh-cn/tutorials/generative/style_transfer.ipynb [hunk 6]
```diff
@@ -0,0 +1,1177 @@
...
+        "colab_type": "text",
+        "id": "Ds4o1h4WHz9U"
+      },
+      "source": [
+        "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\n",
+        "  \u003ctd\u003e\n",
+        "    \u003ca target=\"_blank\" href=\"https://tensorflow.google.cn/tutorials/generative/style_transfer\"\u003e\u003cimg src=\"https://www.tensorflow.google.cn/images/tf_logo_32px.png\" /\u003e在 tensorflow.google.cn 上查看\u003c/a\u003e\n",
+        "  \u003c/td\u003e\n",
+        "  \u003ctd\u003e\n",
+        "    \u003ca target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/generative/style_transfer.ipynb\"\u003e\u003cimg src=\"https://www.tensorflow.google.cn/images/colab_logo_32px.png\" /\u003e在 Google Colab 上运行\u003c/a\u003e\n",
```
Comment (lamberta) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]: No "www." for tensorflow.google.cn

Comment (lamberta) [inline site/zh-cn/tutorials/generative/style_transfer.ipynb]: Make sure to replace "en/" with "zh-cn/" in the GitHub URL

**Pull request merged**